hubspot3/test/test_tickets.py

```diff
@@ -49,3 +60,206 @@ def test_get_all_tickets():
     assert tickets
     assert isinstance(tickets, list)
     assert len(tickets) > 1
+
+
+@pytest.mark.parametrize(
+    "extra_properties_given, extra_properties_as_list",
+    [
+        (None, []),
+        ("lead_source", ["lead_source"]),
+        (
+            ["hs_analytics_last_url", "hs_analytics_revenue"],
+            ["hs_analytics_last_url", "hs_analytics_revenue"],
+        ),
+    ],
+)
+def test_get_batch(
+    tickets_client,
+    mock_connection,
+    extra_properties_given,
+    extra_properties_as_list,
+):
+    response_body = {
+        "3234574": {"objectId": 3234574, "properties": {}},
+        "3234575": {"objectId": 3234575, "properties": {}},
+    }
+    ids = ["3234574", "3234575"]
+    properties = tickets_client.default_batch_properties.copy()
+    properties.extend(extra_properties_as_list)
+    params = {"properties": p for p in properties}
+
+    mock_connection.set_response(200, json.dumps(response_body))
+    resp = tickets_client.get_batch(ids, extra_properties_given)
+    mock_connection.assert_num_requests(1)
+    mock_connection.assert_has_request(
+        "POST", "/crm-objects/v1/objects/tickets/batch-read", **params, data={'ids': ids}
+    )
+    assert len(resp) == 2
+    assert {"id": 3234574} in resp
+    assert {"id": 3234575} in resp
+
+
+@pytest.mark.parametrize(
+    "extra_properties_given, extra_properties_as_list",
+    [
+        (None, []),
+        ("lead_source", ["lead_source"]),
+        (
+            ["hs_analytics_last_url", "hs_analytics_revenue"],
+            ["hs_analytics_last_url", "hs_analytics_revenue"],
+        ),
+    ],
+)
+def test_get_batch_with_history(
+    tickets_client,
+    mock_connection,
+    extra_properties_given,
+    extra_properties_as_list,
+):
+    response_body = {
```

Comment:
This `response_body` appears twice in the code. It can be extracted.

Reply:
Same for the `ids`.

Reply:
I could, but there is no reason for them to be equal, they could be changed for more specific tests, which would create similar variables global and not